### ISSUE TITLE
No Print to Fileunless MakerBot connected whlie RepG starts

### ISSUE THREAD

**Issue by charlespax** [makerbot/ReplicatorG#99]
When using the RepG master branch in git I can't build to file.

```
 [java] Exception in thread "Thread-8" java.lang.AssertionError
 [java]     at replicatorg.drivers.gen3.Sanguino3GDriver.runCommand(Sanguino3GDriver.java:165)
 [java]     at replicatorg.drivers.gen3.Sanguino3GDriver.reconcilePosition(Sanguino3GDriver.java:1063)
 [java]     at replicatorg.drivers.DriverBaseImplementation.getCurrentPosition(DriverBaseImplementation.java:232)
 [java]     at replicatorg.app.MachineController$MachineThread.buildInternal(MachineController.java:312)
 [java]     at replicatorg.app.MachineController$MachineThread.run(MachineController.java:446)
```

**Comment by charlespax** [comment]
Machine is now connected. Print to file works.

**Comment by charlespax** [comment]
I then unplugged the TTL cable from the machine and left it in the
computer. It worked. Then unplugged the TTL cable from the computer,
it worked. Restarted repg in this configuration and tried it, not
working.

**Comment by phooky** [comment]
- Do not assert a serial port if writing to s3g file.  Closed by 04bf8175f439e21a64f4157ab52069ac1dc8001f.
